galaxy/api/v2/tests/test_task_serializers.py

```diff
@@ -0,0 +1,135 @@
+# (c) 2012-2019, Ansible by Red Hat
+#
+# This file is part of Ansible Galaxy
+#
+# Ansible Galaxy is free software: you can redistribute it and/or modify
+# it under the terms of the Apache License as published by
+# the Apache Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# Ansible Galaxy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# Apache License for more details.
+#
+# You should have received a copy of the Apache License
+# along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
+
+from collections import OrderedDict
+from unittest import mock
+import uuid
+
+import pytest
+from django.utils import timezone
+from pulpcore import constants as pulp_const
+from pulpcore.app import models as pulp_models
+from rest_framework import exceptions as drf_exc
+import semantic_version
+
+from galaxy.main import models
+from galaxy.api.v2 import serializers
+
+
+class TestCollectionUploadSerializer:
```

Comment:
Why don't these tests inherit from any `TestCase` class?

Reply:
Because they don't use database access and API client.